test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex routing url
+   Given Minishift has state "Running"
+     When status code of HTTP request to "Routing" is equal to "200"
```

Comment:
It is also about the term "Routing". It really does not make sense in this context. The application exposes a service as a route. That's the terminology really. One cannot make a request to "Routing".